### DISCUSSION TITLE
Einfärbung Oberfläche

### DISCUSSION THREAD

**Discussion by Deanw1975** [solectrus/solectrus#2903] [General]
Hallo zusammen,

ich habe zwei Solectrus Installationen am laufen.
Kann ich irgendwo den blauen oberen Balken einfärben um die Installationen auseinander zu halten. Eine Beschriftung wäre auch Super.
Einfärben würde mir aber schon reichen.

Danke
Dean

**Comment by ledermann** [comment]
Derzeitig ist eine solche Individualisierung nicht vorgesehen. Bei der Kachelansicht "Einzelwerte" lässt sich zwar eine Beschriftung erfassen, diese hat es aber (auch aus Platzgründen) noch nicht in die Strombilanz geschafft. 

Ich stimme dir zu, dass so etwas nützlich wäre. Auch die Idee mit der Einfärbung gefällt mir - der Einfachheit wegen. Für das Thema "Farben" gibt es hier ein Issue, allerdings noch keinen Plan, das kurzfristig anzugehen:
https://github.com/solectrus/solectrus/issues/1859
